lstchain/io/event_selection.py

```diff
@@ -88,6 +100,12 @@ class DL3Cuts(Component):
         default_value=0.32,
     ).tag(config=True)
 
+    max_theta_range = Float(
+        help="Maximum value of theta (deg), approximate for the PSF size," +
+            " to calculate the percentile cut in each energy bin.",
+        default_value=0.7,
```

Comment:
why 0.7?

Reply:
It is used in the Performance paper, and I so kept this value as default/nominal. Maybe each analyzer can select a different one for the specific data-MC comparison.

Reply:
The first option, use the 0.7 deg max theta value when evaluating and applying an efficiency cut